build/nuke/Build.cs

```diff
@@ -64,7 +66,7 @@ void DeleteReparsePoints(string path)
 
     Target Workflow => _ => _
         .Description("GitHub workflow entry point")
-        .After(Clean)
+        .DependsOn(Clean)
```

Comment:
What is the practical difference of changing from `After` to `DependsOn`?

Reply:
thanks to this change `dotnet nuke Workflow` executes `Clean` which is handy for me